src/lib/Microsoft.Health.Fhir.Ingest/Telemetry/Metrics/IomtMetrics.cs

```diff
@@ -159,30 +151,14 @@ public static Metric FhirResourceSaved(ResourceType resourceType, ResourceOperat
         public static Metric UnhandledException(string exceptionName, string connectorStage)
         {
             EnsureArg.IsNotNull(exceptionName);
-            return new Metric(
-                "UnhandledException",
-                new Dictionary<string, object>
-                {
-                    { _nameDimension, exceptionName },
-                    { _categoryDimension, Category.Errors },
-                    { _errorTypeDimension, ErrorType.GeneralError },
-                    { _errorSeverityDimension, ErrorSeverity.Critical },
-                    { _operationDimension, connectorStage },
-                });
+            var metricName = "UnhandledException";
+            return metricName.ToErrorMetric(connectorStage, ErrorType.GeneralError, ErrorSeverity.Critical, errorName: exceptionName);
         }
 
         public static Metric HandledException(string exceptionName, string connectorStage)
         {
-            return new Metric(
-                exceptionName,
-                new Dictionary<string, object>
-                {
-                    { _nameDimension, exceptionName },
-                    { _categoryDimension, Category.Errors },
-                    { _errorTypeDimension, ErrorType.GeneralError },
-                    { _errorSeverityDimension, ErrorSeverity.Critical },
-                    { _operationDimension, connectorStage },
-                });
+            EnsureArg.IsNotNull(exceptionName);
```

Comment:
While it can be called on null, it wouldn't make sense to allow such a call since the metric name would be null and therefore invalid. 
Since we do check in ToErrorMetric if the metric name is null/whitespace, I guess its a redundant check so can remove here, although the exception message will slightly vary where now it might say (metricName is null instead of exceptionName is null).
Having said that, I highly doubt we will ever run into an exceptionName is null (ex.GetType().Name).